README.md

```diff
@@ -0,0 +1,86 @@
+# Teacher Services Cloud
+
+A repo for building Teacher Sevices cloud infrastructure
+
+## Directory Layout
+
+```
+-> cluster
+    -> config
+        *.sh files for each cluster environment
+    -> terraform
+        *.tf files for cluster build
+        -> config
+            *.json files for each cluster environment
+-> custom_domains
+    -> config
+        *.sh for each cluster DNS zone
+    -> terraform
+        -> infrastructure
+            *.tf files for cluster DNS zone build
+            -> workspace_variables
+                *.tfvars, *.json for each cluster DNS zone
+Makefile
+```
+
+## Cluster Configuration
+
+### Cluster Build
+
+Development
+
+make development terraform-{plan/apply} ENVIRONMENT=cluster{n}
+
+    - where n = 1-6
+
+e.g. make development terraform-plan ENVIRONMENT=cluster1
+
+### DNS records
+
+We use wildcard A records for the cluster DNS.
+
+For the development clusters, on cluster build a wildcard DNS record will be automatically created in the dev DNS zone.
+This occurs if cluster_dns_zone is set in the cluster config tfvars.json for that environment.
+
+### Cluster TLS
+
+We use a wilcard TLS certificate for each cluster.
```

Comment:
This is a wildcard certificate for the default domain of any application deployed to the cluster